src/components/DpMultiselect/DpMultiselect.vue

```diff
@@ -41,13 +41,13 @@
       @tag="newVal => $emit('tag', newVal)">
       <template v-slot:noOptions>
         <slot name="noOptions">
-          {{ Translator.trans('explanation.noentries') }}
+          {{ translations.explanationNoentries }}
```

Comment:
I know this key existed already, but while you're at it, I think we should change it to `explanationNoEntries` or better yet, `noEntriesAvailable`

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/918/commits/e3d3f2bda3591511d107c869d9280fea54341ffa
Thanks!